cookbooks/aws-parallelcluster-install/recipes/cluster_admin_user.rb

```diff
@@ -15,20 +15,4 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
Why did you decide to keep this one?

Reply:
I'll remove it, simple I did not completed the move into the base recipe